Applied Challenge 5/Menu.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Menu</title>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="_css/Mowen_a5.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+    <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+    <link rel="stylesheet" type="text/css" href="_css/bistro.css">
+</head>
+<body>
+<!-- Randall Mowen
+     Reservation.html
+     INFO2439
+     Thoendel
+     10/12/19
+-->
+<div data-role="page" id="homePage">
+    <header>
```

Comment:
I love that you included all the pages of the menu in this one file. Very clean!